package.json

```diff
@@ -55,8 +55,8 @@
     "eslint-plugin-flowtype": "^8.0.3",
     "flow-bin": "^0.108.0",
     "mocha": "^10.1.0",
-    "mongojs": "^3.0.0",
-    "nyc": "^14.1.1",
+    "mongojs": "^3.1.0",
+    "nyc": "^15.1.0",
```

Comment:
This removes the annoying handlebars dependency and doesn't have any relevant breaking changes for this lib.